generated/nidcpower/session.py

```diff
@@ -3407,7 +3409,7 @@ def _set_attribute_vi_string(self, attribute_id, attribute_value):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
-    def set_sequence(self, values, source_delays, size):
+    def set_sequence(self, source_delays, size, values=None):
```

Comment:
size should not be here

Reply:
This will be resolved by the len feature @texasaggie97 is working on.